Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM mcr.microsoft.com/dotnet/core/aspnet:2.2
+
+ENV SOLUTION_DLL="WebApi.dll"
+
+WORKDIR /home_dir
+
+COPY  ./artifacts/. .
```

Comment:
what if my directory name is not artifacts?

Reply:
all artifacts are created in the "artifacts" folder for all solutions
 -o ../artifacts